File: modul_praktikum_python/contoh_code/pascal/baca_tulis.pas
uses crt; 
var 
  i : integer; 
  j : real; 
  kar : char; 
  NString : string; 

begin 
  clrscr; 
  writeln(‘Contoh membaca dan menulis’);  
  write(‘Masukkan nilai integer  : ’);readln(i); 
  write(‘Masukkan nilai real     : ‘);readln(j); 
  write(‘Masukkan nilai karakter : ‘);readln(kar); 
  write(‘Masukkan nilai string   : ‘);readln(Nstring); 
  writeln(‘Nilai integer yang dibaca  =‘,i); 
  writeln(‘Nilai real yang dibaca     =‘,j); 
  writeln(‘Nilai karakter yang dibaca =‘,kar); 
  writeln(‘Nilai string yang dibaca   =‘,NString); 
  readkey;  
end.

File: modul_praktikum_python/contoh_code/python/baca_tulis.py
"""Contoh membaca dan menulis beberapa tipe dari input."""


def main() -> None:
    print("Contoh membaca dan menulis")
    i = int(input("Masukkan nilai integer  : "))
    j = float(input("Masukkan nilai real     : "))
    kar = input("Masukkan nilai karakter : ").strip()[:1] or " "
    n_string = input("Masukkan nilai string   : ")
    print("Nilai integer yang dibaca  =", i)
    print("Nilai real yang dibaca     =", j)
    print("Nilai karakter yang dibaca =", kar)
    print("Nilai string yang dibaca   =", n_string)
    input("Tekan Enter untuk melanjutkan...")


if __name__ == "__main__":
    main()
